PrBuild/config.json

```diff
@@ -1,12 +1,14 @@
-{      
-    "repo_owner": "pniko",
-    "repo_name": "detox-appcenter",
-
-    "appcenter_owner_type": "users",
-    "appcenter_owner": "pniko",
-    "appcenter_app": "detox-appcenter",
-
-    "branch_template": "master",
-    "branch_regex": "(skip|no-build)"
+{
+    "repos": [
```

Comment:
Perfect on the adjustment to an array here.

Reply:
Could you please clarify what do you mean by this? Should I leave it as it is or replace file contents with an array like this:
```json
[
    {
         "provider": "github",
         ...
    }, 
    ...
]
```